ci-operator/step-registry/service-registry/run-tests/service-registry-run-tests-commands.sh

```diff
@@ -4,14 +4,18 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
-echo "Run the tests"
-./scripts/run-interop-tests.sh
+function copyArtifacts() {
+  echo "Rename JUnit files"
+  rename TEST junit_TEST target/surefire-reports/*.xml
+
+  echo "Remove unnecessary comma in decimal numbers in JUnit files"
+  sed -i -r 's/([0-9]),([0-9])/\1\2/g' target/surefire-reports/*.xml
 
-echo "Rename JUnit files"
-rename TEST junit_TEST target/surefire-reports/*.xml
+  echo "Copy JUnit files into artifacts dir"
+  cp target/surefire-reports/*.xml "${ARTIFACT_DIR}"
+}
 
-echo "Remove unnecessary comma in decimal numbers in JUnit files"
-sed -i -r 's/([0-9]),([0-9])/\1\2/g' target/surefire-reports/*.xml
+trap copyArtifacts EXIT
```

Comment:
```suggestion
trap copyArtifacts SIGINT SIGTERM ERR EXIT
```
Not saying what you have is wrong, because I genuinely don't know if using just `EXIT` works for all scenarios. i used the trap above in my scenarios, just to be sure. If you want to keep it as `EXIT`, that's fine with me.